app/views/claims/schools/claims/show.html.erb

```diff
@@ -5,29 +5,60 @@
 <% end %>
 
 <div class="govuk-width-container">
-  <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
-  <h2 class="govuk-heading-m"><%= t(".providers_heading") %></h2>
-
-  <%= govuk_task_list(id_prefix: "providers", classes: "providers-task-list") do |task_list| %>
-    <%= task_list.with_item(
-      title: t(".itt_provider"),
-      href: edit_claims_school_claim_path(@school, @claim, "providers"),
-      status: govuk_tag(
-        text: @claim.item_status_tag("provider").fetch(:text),
-        colour: @claim.item_status_tag("provider").fetch(:colour),
-      ),
-    ) %>
-  <% end %>
-
-  <h2 class="govuk-heading-m"><%= t(".mentors_heading") %></h2>
-  <%= govuk_task_list(id_prefix: "mentors", classes: "mentors-task-list") do |task_list| %>
-    <%= task_list.with_item(
-      title: t(".general_mentors"),
-      href: edit_claims_school_claim_path(@school, @claim, "mentors"),
-      status: govuk_tag(
-        text: @claim.item_status_tag("mentors").fetch(:text),
-        colour: @claim.item_status_tag("mentors").fetch(:colour),
-      ),
-    ) %>
-  <% end %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= t(".heading") %> <%= @claim.reference %></h1>
```

Comment:
Should we rename this to `page_title` and pass the reference to the translation? This way we can use it for the page title as well, which we should add as the page doesn't have a page title.


```suggestion
      <h1 class="govuk-heading-l"><%= t(".page_title" reference: @claim.reference) %></h1>
```